core/sessions/user.go

```diff
@@ -18,13 +18,21 @@ import (
 type User struct {
 	Email             string
 	HashedPassword    string
+	Role              string
```

Comment:
I would ask to create a custom type for this:

```
type Role string
const (
UserRoleAdmin = Role("admin")
...
)
```

Reply:
Updated, PTAL 🙂